mesonbuild/dependencies/base.py

```diff
@@ -459,7 +459,10 @@ def _set_cargs(self):
     def _set_libs(self):
         env = None
         libcmd = [self.name, '--libs']
-        if self.static:
+        try_static = self.static or \
+                     self.env.coredata.get_builtin_option('default_link') == 'static'
```

Comment:
__[Flake8]__

continuation line over-indented for visual indent


<!-- Comment by [SideCI](https://sideci.com) -->